### PR TITLE
enhance(job): simplify job eval cmd 

### PR DESCRIPTION
**Pull request by goldenxinxing**
## Description

`swcli model eval ` optimize：
1. delete "src_dir"
2. simplify param: "type"
3. generate “eval_job.yaml”  if current work dir haven't this yaml file.
image

4. eval result can be searched with cmd：`swcli eval list`
image

## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [x] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

**Comment by tianweidut** [comment]
@goldenxinxing  pretty output job info ?

File: client/setup.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def _format_version() -> str:
     author_email="developer@starwhale.ai",
     version=_format_version(),
     description="An MLOps Platform for Model Evaluation",
-    long_description=open("../README.md", encoding="utf-8").read(),
+    long_description=open("./README.md", encoding="utf-8").read(),
```
Comment (tianweidut) [inline client/setup.py:50]: it is the wrong path, please use `../README.md`.
does it modify by IDE automatically?

**Comment by tianweidut** [comment]
Some dir name advises:
- `${workdir}/eval_jobs.yaml` -> `${workdir}/runlog/eval_jobs.yaml` 
- `${workdir}/runlog` -> ${workdir}/.runlog`
- create a `.gitignore` file in the ${workdir}/.runlog` dir,  use [make_dir_gitignore](https://github.com/star-whale/starwhale/blob/main/client/starwhale/utils/__init__.py#L192) function.

- `eval_jobs.yaml` is under the hood, not for user perspective.
- hide runlog that minimizes the disruption of users' operations.
- if runlog is in the user source code dir, ignoring it by git automatically is recommended, just like venv.

cc @goldenxinxing

File: client/starwhale/core/model/model.py [hunk 2]
```diff
@@ -230,20 +241,23 @@ def eval_user_handler(
         if job_name not in _jobs:
             raise RuntimeError(f"job:{job_name} not found")
 
+        project = os.getenv(SWEnv.project)
+        if project is None:
+            project = SWCliConfigMixed().current_project
```
Comment (tianweidut) [inline client/starwhale/core/model/model.py:246]: in the cloud task container, what is the current_project?

File: client/starwhale/core/model/model.py [hunk 3]
```diff
@@ -215,13 +213,26 @@ def eval_user_handler(
...
+        _model_config = cls.load_model_config(workdir / model_yaml_name)
+        if not version:
+            version = gen_uniq_version()
+        if _model_config.run.typ == EvalHandlerType.DEFAULT:
+            _module = DEFAULT_EVALUATION_PIPELINE
+        else:
+            _module = _model_config.run.ppl
+
+        _yaml_path = str(workdir / DEFAULT_EVALUATION_JOBS_FNAME)
+        #
```
Comment (tianweidut) [inline client/starwhale/core/model/model.py:225]: remove this useless comment?

**Review by tianweidut — CHANGES_REQUESTED** [review]

File: client/setup.py (hunk 1 above)
Comment (goldenxinxing) [reply client/setup.py]: done

File: client/starwhale/core/model/model.py (hunk 3 above)
Comment (goldenxinxing) [reply client/starwhale/core/model/model.py:225]: done

**Comment by goldenxinxing** [comment]
> deployed

now use project/evaluation as run dir，no longer in the current dir.

**Comment by goldenxinxing** [comment]
> Some dir name advises:
> 
> * `${workdir}/eval_jobs.yaml` -> `${workdir}/runlog/eval_jobs.yaml`
> * `${workdir}/runlog` -> ${workdir}/.runlog`
> * create a `.gitignore` file in the ${workdir}/.runlog` dir,  use [make_dir_gitignore](https://github.com/star-whale/starwhale/blob/main/client/starwhale/utils/__init__.py#L192) function.
> * `eval_jobs.yaml` is under the hood, not for user perspective.
> * hide runlog that minimizes the disruption of users' operations.
> * if runlog is in the user source code dir, ignoring it by git automatically is recommended, just like venv.
> 
> cc @goldenxinxing

now use project/evaluation as run dir，no longer in the current dir.

File: client/starwhale/core/model/model.py (hunk 2 above)
Comment (goldenxinxing) [reply client/starwhale/core/model/model.py]: First, use Project URI to parse, so, if a user sends or sets env for the project, use it. if exec without --project, use current project(get from starwhale config)

**Review by tianweidut — APPROVED** [review]
LGTM

**Pull request merged**